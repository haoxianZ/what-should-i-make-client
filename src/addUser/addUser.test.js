import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AddUser from './addUser'

describe(`sign up form component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<AddUser />);
    expect(toJson(wrapper)).toMatchSnapshot()
  })
}) 