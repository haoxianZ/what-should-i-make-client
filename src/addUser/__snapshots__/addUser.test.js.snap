// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign up form component renders the complete form 1`] = `
<div>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <form
    className="login"
    onSubmit={[Function]}
  >
    <label
      htmlFor="username"
    >
      Username:
    </label>
    <input
      id="username"
      name="username"
      required={true}
      type="text"
    />
    <br />
    <br />
    <label
      htmlFor="email"
    >
      Email:
    </label>
    <input
      id="email"
      name="email"
      required={true}
      type="email"
    />
    <br />
    <br />
    <label
      htmlFor="password"
    >
      Password:
    </label>
    <input
      id="password"
      name="password"
      required={true}
      type="password"
    />
    <br />
    <br />
    <label
      htmlFor="confirmPassword"
    >
      Confirm Your Password:
    </label>
    <input
      id="confirmPassword"
      name="confirmPassword"
      required={true}
      type="password"
    />
    <button
      type="submit"
    >
      Sign up
    </button>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/"
    >
      <button>
        Cancel
      </button>
    </Link>
  </form>
</div>
`;
